contentcuration/contentcuration/static/js/edit_channel/share/views.js

```diff
@@ -44,9 +47,20 @@ var ShareView = BaseViews.BaseView.extend({
     render: function() {
         this.$el.html(this.template({
             channel:this.model.toJSON(),
-            user: this.current_user.toJSON()
+            user: this.current_user.toJSON(),
+            share_modes: this.get_share_modes()
         }));
     },
+    get_share_modes: function(){
+        if (!this.share_modes){
+            if(_.find(window.current_channel.get("editors"), function(u){return u === this.current_user.id})){
```

Comment:
It'd be a lot easier to understand the condition here if `_.find` was set as a boolean outside of the if statement with formatting